documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -105,3 +110,16 @@ kubectl create secret generic _CA-KEY-SECRET_ --from-file=ca.key=_CA-KEY-SECRET-
 kubectl label secret _CA-CERTIFICATE-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_KAFKA-CLUSTER-NAME_
 kubectl label secret _CA-KEY-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_KAFKA-CLUSTER-NAME_
 ----
+
+. Annotate the secrets with the generation annotations:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-CERTIFICATE-SECRET_ strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_
+kubectl annotate secret _CA-KEY-SECRET_ strimzi.io/ca-key-generation=_CA-KEY-GENERATION_
+----
++
+When the Strimzi generated CA certificates are used, the generation increment is automatically handled by the cluster operator.
+In this case, with manual renewal of your own CA certificates, you have to take care to increment the generation on renewal accordingly.
+The generation annotations have to have the next incremental value of the current ones from the Secret.
```

Comment:
Should it be the next value or just higher value? I wonder if it could cause some confusion when you for example due to some errors try to do another renewal before the first one finished etc.

Reply:
Good point. It has to be an higher value, no need to be exactly the next one.